pkg/innerring/innerring.go

```diff
@@ -72,8 +72,8 @@ type (
 		// event producers
 		morphListener   event.Listener
 		mainnetListener event.Listener
-		blockTimers     []*timer.BlockTimer
 		epochTimer      *timer.BlockTimer
+		initEpochTimer  atomic.Pointer[timer.BlockTimer]
```

Comment:
Is it still `init`? Looks like it's the only one left.

Reply:
didnt get what u mean. This is the `initialEpochTicker` revealed, previously it was in `blockTimers` slice

Reply:
OK, looks like I've not noticed the other one.